index.ts

```diff
@@ -12,7 +12,7 @@ const img = new Image;
 const ac = new (typeof webkitAudioContext !== 'undefined' ? webkitAudioContext : AudioContext)();
 
 const canvas = document.querySelector('canvas') as HTMLCanvasElement;
-const velocity = { r: 0, rotationVelocity: 0, maxVelocity: 100 };
+const velocity = { r: 0, rotationVelocity: 0, maxVelocity: 10 };
```

Comment:
looking at what the actual velocity values were.. they are pretty much always 0-6.